airbnb/src/components/PriceRange/PriceRange.tsx

```diff
@@ -0,0 +1,137 @@
+import { useEffect, useState } from 'react';
+import BarChart from './Canvas';
+import { Box, Slider } from '@mui/material';
+import {
+  PriceRangeContainer,
+  PriceRangeTitle,
+  PriceRangePrice,
+  PriceRangeCaption,
+} from './PriceRange.styled';
+const sampleData = [

```

Comment:
여기서 배열 타입을 정의해주시고 밑에서 any사용을  안 하시면 좋을 것 같습니다.
```suggestion
type PriceInfo ...

const sampleData: Array<PriceInfo> = [
 ...
```